src/plugins/DevTools/components/ActionsComponent.ts

```diff
@@ -1,36 +1,106 @@
-import { ActionPayload } from '../../../core/types/Flux';
+import { ActionPayload, ActionHandler, Action } from '../../../core/types/Flux';
 import { OwlUIComponent } from '../../../ui/JWOwlUIPlugin';
+import { HandlerToken, DispatcherRegistryRecord } from '../../../core/dispatcher/Dispatcher';
 
 interface ActionsState {
-    selectedIndex: number;
+    actions: Action[]; // Stack of all actions performed since init
+    openTab: 'selected' | 'registry';
+    registry: DispatcherRegistryRecord[];
+    selectedActionIndex: number;
+    selectedHandlerToken: HandlerToken | null;
```

Comment:
I guess you're right. But isn't it more explicit to specify that it can be `null` though? So we know we should test for that? I think I'd rather add ` | null` next to `selectedActionIndex: number`.